src/main/java/com/eleks/academy/whoami/model/response/HistoryDetails.java

```diff
@@ -0,0 +1,25 @@
+package com.eleks.academy.whoami.model.response;
+
+import com.eleks.academy.whoami.repository.HistoryChat;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Data
+@NoArgsConstructor
+public class HistoryDetails {
```

Comment:
I don't fully get the purpose of this Class, if it is only to show string representation of HisctoryChat then why use whole new class, put it in the map and then just get it from Map and not just override toString in HistoryChat?
Is there some other purpose of this class?